contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -44,9 +47,20 @@ var ShareView = BaseViews.BaseView.extend({
     render: function() {
         this.$el.html(this.template({
             channel:this.model.toJSON(),
-            user: this.current_user.toJSON()
+            user: this.current_user.toJSON(),
```

Comment:
I would expect all of the data to be included inside of `this.model`. Is there a reason this isn't the case?

I can see an argument for trying to keep models the same structure as what the API returns. Is that what we're going for?